dl/src/main/scala/com/intel/analytics/bigdl/optim/Predictor.scala

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
should we put this in another package(e.g. predict)? It's not related with optim. @jason-dai